actioncable/lib/action_cable/helpers/action_cable_helper.rb

```diff
@@ -20,9 +20,16 @@ module ActionCableHelper
       # Make sure to specify the correct server location in each of your environments
       # config file:
       #
-      #   config.action_cable.url = "ws://example.com:28080"
+      #   config.action_cable.mount_path = "/cable123"
+      #   <%= action_cable_meta_tag %> would render:
+      #   => <meta name="action-cable-url" content="/cable123" />
+      #
+      #
       def action_cable_meta_tag
-        tag "meta", name: "action-cable-url", content: Rails.application.config.action_cable.url
+        tag "meta", name: "action-cable-url", content: (
+          ActionCable.server.config.mount_path ||
```

Comment:
By @matthewd's comments this should be `ActionCable.server.config.url || ActionCable.server.config.mount_path || raise(...)`